Gateway/Response/PayLinkHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Buckaroo\Magento2\Gateway\Response;
+
+use Buckaroo\Transaction\Response\TransactionResponse;
+use Magento\Payment\Gateway\Data\PaymentDataObjectInterface;
+use Magento\Payment\Gateway\Response\HandlerInterface;
+use Magento\Framework\Event\ManagerInterface;
+
+class PayLinkHandler extends AbstractResponseHandler implements HandlerInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function handle(array $handlingSubject, array $response)
+    {
+        $this->validate($handlingSubject, $response);
+
+        $response = $response['object'];
+        $paylink = $response->getServiceParameters()['paylink'] ?? '';
+
+        if (!empty($paylink)) {
+            $this->messageManager->addSuccess(
+                __(
+                    'You PayLink <a href="%1">%1</a>',
```

Comment:
Fix typo here to 'Your'

Reply:
"You Bad" :)) 
I copied from this app/code/Buckaroo/Magento2/Controller/Adminhtml/PayLink/Index.php:147